pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -240,6 +240,13 @@ func (pl *CSILimits) Filter(ctx context.Context, _ *framework.CycleState, pod *v
 		return nil
 	}
 
+	for _, driverName := range newVolumes {
+		if !pl.checkCSIDriverOnNode(driverName, csiNode) {
+			driverNotInstalledMsg := fmt.Sprintf("%s CSI driver is not installed on the node", driverName)
+			return framework.NewStatus(framework.Unschedulable, driverNotInstalledMsg)
```

Comment:
How does this impact pod priority? Isn't it will increase scheduling latency?

Reply:
It could, but then it also could speed up scheduling if cluster has nodes which have CSI driver installed and scheduler skipped past those to send pod on a node that does not even have CSI driver.